html/project.sample/project_specific_prefs.inc

```diff
@@ -57,6 +59,8 @@ define('APP_SELECT_DESC', tra('Run only the selected applications'));
 define('APP_SELECT_TOOLTIP', tra('Only get tasks for certain applications. Useful to focus on particular applications, or to exclude them.'));
 define('ACCEPT_ANY_DESC', tra('If no work for selected applications is available, accept work from other applications?'));
 define('NON_GRAPHICAL_DESC', tra("Use faster non-graphical applications if available?"));
+define("LIMIT_TASKS_DESC", tra("Limit the number of tasks per host?", "<br><span class=note>", "</span>"));
```

Comment:
The two extra options are not needed since there is no `%1` and `%2` in the first argument. something like that would work tho:
`define("LIMIT_TASKS_DESC", tra("Limit the number of tasks per host? %1Description of this feature%2", "<br><span class=note>", "</span>"));`